src/linkml/include_participant.yaml

```diff
@@ -147,16 +156,21 @@ slots:
     title: Family Type
     range: enum_family_type
     required: true
-  father_id:
-    definition_uri: include:father_id
-    description: Participant ID for Participant's father
-    title: Father Id
-    range: string
+  relationship_type:
```

Comment:
I might name this family_relationship_type  - since it specifically holds familial relationships.